### PR TITLE
Change log level for task_enqueued & receiver_task_success

### PR DESCRIPTION
**Pull request by areski**
On my projects I have a large amount tasks, therefore all the messages `task_enqueued` & `task_succeeded` are producing a large volume of logs, which make the log file very hard to follow/read. 
I feel that those message should be at a different log level.

I could not find a way to change this directly on my project without changing `django_structlog`, 
if there is neater way please let me know.

Thanks!

### CODE REVIEW

**Comment by jrobichaud** [comment]
Can you try with a python logging filter? Its supposed to be used for this kind of use case.

https://stackoverflow.com/a/879937/2115513

Please reopen if there is a problem in the library that prevent you to do this.

Thanks.

**Comment by jrobichaud** [comment]
Maybe you need to use structlog's filtering processor in this case:

https://www.structlog.org/en/stable/processors.html#filtering

**Comment by jrobichaud** [comment]
I am interested in the final solution you will choose, I could add something about this in the documentation.

**Comment by areski** [comment]
Thanks so much for your reply, much appreciated.

I ended up using processors filtering.

As it might be useful for other so here the code snippet:

```
from structlog import DropEvent

class ConditionalDropper:
    def __init__(self, event_to_ignore):
        self._event_to_ignore = event_to_ignore

    def __call__(self, logger, method_name, event_dict):
        """
        >>> cd = ConditionalDropper(["task_enqueued", "task_succeeded"])
        >>> cd(None, None, {"event": "task_failed", "msg": "custom message"})
        {'msg': 'custom message', 'event': 'task_failed'}
        >>> cd(None, None, {"event": "task_enqueued", "msg": "custom message"})
        Traceback (most recent call last):
        ...
        DropEvent
        """
        if event_dict.get("event") in self._event_to_ignore:
            raise DropEvent
        else:
            return event_dict
```

Then where you configure structlog for Celery:
```
    structlog.configure(
        processors=[
            ConditionalDropper(["task_enqueued", "task_succeeded"]),
            structlog.contextvars.merge_contextvars,
            ...
```

We could still have a conversation about which loglevel is appropriate for some event, I feel that some should be in `debug` rather than `info`, rather that filtering them out completely it would be nice to set a different one.

Huge thanks for your help!

**Comment by jrobichaud** [comment]
Thanks @areski. 

For the moment I will leave them as-is since you are the first to do this observation. 

If more people observe the same thing I will consider changing some log levels.
